homeassistant/components/sensor/gtfs.py

```diff
@@ -219,11 +223,16 @@ def icon(self):
         """Icon to use in the frontend, if any."""
         return ICON
 
+    @property
```

Comment:
This component actually already implements `device_state_attributes`, which returns `self._attributes`. `self._attributes` is updated inside `update`, so just make sure you set the attribute there.